### PR TITLE
Update BSPD/Brakelight code to use new libraries

### PR DESCRIPTION
**Pull request by jack-greenberg**

### CODE REVIEW

File: vehicle/mkv/software/brakelight_bspd/bspd.c [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#include "libs/adc/api.h"
+#include "libs/can/api.h"
+#include "libs/timer/api.h"
+#include <avr/interrupt.h>
```
Comment (mpatil99) [inline vehicle/mkv/software/brakelight_bspd/bspd.c:4]: Hmm do we still need an avr reference here? Feels like this was kinda the point of the lib rewrite?
Actually looking at: https://www.nongnu.org/avr-libc/user-manual/group__avr__interrupts.html, looks sei, cei come from here. 
I wonder if in the future we want to move these to the interrupt library as to remove the need for avr specific imports in app code

File: vehicle/mkv/software/brakelight_bspd/config.h [hunk 2]
```diff
@@ -0,0 +1,51 @@
+#include "libs/adc/api.h"
```
Comment (mpatil99) [inline vehicle/mkv/software/brakelight_bspd/config.h:1]: Nit/question, is worth renaming this file to bspd_config.h purely in an effort to change the line in the BUILD file to smthg more specific?

File: vehicle/mkv/software/brakelight_bspd/BUILD [hunk 3]
```diff
@@ -0,0 +1,19 @@
+load("//bazel:defs.bzl", "cc_firmware")
+
+cc_firmware(
+    name = "bspd",
+    srcs = [
+        "bspd.c",
+        "config.h",
```
Comment (mpatil99) [inline vehicle/mkv/software/brakelight_bspd/BUILD:7]: With reference to my other comment, this line would become bspd_config.h

File: vehicle/mkv/software/brakelight_bspd/config.h [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        .pin_behavior = DISCONNECTED,
+        .interrupt_enable = true,
+        .interrupt_callback = timer0_callback,
+    },
+};
+
+/*
+ * CAN messages
+ */
+#define CAN_MSG_DLC (5)
```
Comment (mpatil99) [inline vehicle/mkv/software/brakelight_bspd/config.h:44]: Cool!

**Review by mpatil99 — APPROVED** [review]
Some general nits. Approved, if you decide to ignore/punt them

File: vehicle/mkv/software/brakelight_bspd/bspd.c (hunk 1 above)
Comment (jack-greenberg) [reply vehicle/mkv/software/brakelight_bspd/bspd.c:4]: Yeah I've thought about this a bit... I think I'm going to keep it as is for now, and we can revisit down the line if we want.

File: vehicle/mkv/software/brakelight_bspd/config.h (hunk 2 above)
Comment (jack-greenberg) [reply vehicle/mkv/software/brakelight_bspd/config.h:1]: Good idea!

**Review by mpatil99 — APPROVED** [review]

**Review by mpatil99 — APPROVED** [review]

**Pull request merged**